### ISSUE TITLE
Search highlights matching part of table names but doesn't narrow down the list

### ISSUE THREAD

**Issue by jsynowiec** [jakob/Postico#508]
## What did you do?

- I connected to my database,
- I've typed `ten` into the search field,

## What did you expect to happen?

- To see only those tables that have `tent` as a part of their name,
- To see `ten` part of tables names highlighted,

## What actually happened?

- Only the `ten` part of table names was highlighted,
- List wasn't narrowed to matching tables,

## What software versions are you using?

Postico version: 1.3.2 (2318)

macOS version: 10.12.6 (16G1212)

PostgreSQL version: 9.5.10

**Comment by jakob** [comment]
Hm. Could it be that the database name contains "ten"? There's a bug in Postico where tables are not filtered when the database name contains the filter string.

**Comment by jsynowiec** [comment]
Yes. The database name contains the _ten_ string. Thanks for the update.
